cards.js

```diff
@@ -16,8 +16,124 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
+function buildDeck(suits, values){
+  var deck = [];
+  for(i in suits) {
+    for(j in values) {
+      deck.push({suit: suits[i], value: values[j]});
+    }
+  }
+  return deck;
+}
+
+newDeck = buildDeck(suits, values);
+newDeck = shuffle(newDeck);
+
+var namePlayers = function() {
+  var player1 = "Joe";
+  var player2 = "Frederico";
+  return [player1, player2];
+}
+
+
+var highCard = function(deck, namePlayers, values) {
+  players = namePlayers();
+  playerCards = [];
```

Comment:
I love where your heads out, and honestly, its not wrong to do this. I imagine that it might save you some time to just code an array of strings here instead of defining a function on lines 32-36 to do that. My question is, going forward, what do you see the namePlayers function doing going forward, besides instantiating an array of strings for you?